### PR TITLE
Add support for EDITOR env variable

### PR DESCRIPTION
**Pull request by mchataigner**
Instead of saving the editor to a config file, this will try to use EDITOR env variable if it is set to a correct value.

### CODE REVIEW

**Comment by simeg** [comment]
Hi @mchataigner, and thanks for contributing! This is a great idea. Me being a rookie rustacean will have to take some time to understand the changes, especially since this project doesn't have any tests yet.

File: src/main.rs [hunk 1]
```diff
@@ -153,8 +159,28 @@ fn get_commit_msg() -> String {
     input
 }
 
-fn open_editor(bin_path: &String, file_path: &String) -> Result<(), Error> {
-    match Command::new(bin_path).arg(file_path).status() {
+fn find_bin(editor: String) -> Option<(String, Vec<String>)> {
```
Comment (simeg) [inline src/main.rs:162]: Why does this has to return a tuple? Isn't it enough with `Option<String>`?

**Comment by mchataigner** [comment]
The idea is to be able to pass some arguments to your editor. e.g. my EDITOR is `emacsclient -t -d emacsdaemon`. Hence the `(string, vec<string>)`

**Comment by xeruf** [comment]
Please merge this :)

**Comment by simeg** [comment]
@mchataigner I don't understand why the `$EDITOR` variable cannot be used as is, even with parameters. Because of this I created another PR (#61) to solve it so I'm closing this.

Still curious so if you want to explain I'd appreciate it!

**Comment by xeruf** [comment]
You linked to the wrong PR ;) it's #61

**Comment by simeg** [comment]
Hehe typing is hard 😄 thanks @xerus2000

**Pull request closed without merging**